.github/workflows/release.yml

```
@@ -0,0 +1,69 @@
+    - name: Run tests
+      run: poetry run pytest tests/unit/ -v
+
+    - name: Run linting
+      run: poetry run ruff check .
+
+    - name: Run type checking
+      run: poetry run mypy src/alphaloop_core/ --ignore-missing-imports --disable-error-code=misc --disable-error-code=unused-ignore --disable-error-code=unreachable --disable-error-code=no-any-return --disable-error-code=dict-item --disable-error-code=attr-defined --disable-error-code=index --disable-error-code=var-annotated --disable-error-code=operator --disable-error-code=call-overload --disable-error-code=union-attr --disable-error-code=no-untyped-def --disable-error-code=safe-super
+
+    - name: Generate release notes
+      id: release_notes
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2001:style:8:19: See if you can use ${variable//search/replace} instead

(shellcheck)